buildpacks/jvm/CHANGELOG.md

```diff
@@ -12,6 +12,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Buildpack API version changed from `0.9` to `0.10`, and so requires `lifecycle` `0.17.x` or newer. ([#662](https://github.com/heroku/buildpacks-jvm/pull/662))
 - This buildpack is no longer Heroku stack specific and can be used with most x86 Linux based CNB build and run images. ([#662](https://github.com/heroku/buildpacks-jvm/pull/662))
 - Default OpenJDK distribution is now always Azul® Zulu®. ([#662](https://github.com/heroku/buildpacks-jvm/pull/662))
+- Some error messages relating to OpenJDK installation changed. ([#665](https://github.com/heroku/buildpacks-jvm/pull/665))
```

Comment:
(thought for the future, not for this PR) The fact that user facing error messages changed and zero integration tests needed updating suggests there isn't any integration testing of these error scenarios. 

Whilst we obviously want to avoid integration tests being too tightly coupled to the things that will cause test churn, I think error messages are such an important part of the UX that it's worth having tests for them, plus unit tests don't cover things like "if the user's app has a broken config, do earlier stages of the buildpack fall over and mean we don't actually get to our expected error message after all", or "if there were several errors or warnings, which do we just show the error or show the warnings too?" etc. 

Thankfully integration tests for failure modes tend to be very quick to run (if the buildpack does the right thing for end-user UX and fails early for broken app configs etc), and so such tests tend to have minimal impact on test end to end time.

For example here are some of the failure cases I test for Python:
https://github.com/heroku/buildpacks-python/blob/de378063c83e1eec6302e79f93f189162b5bbbbf/tests/package_manager_test.rs#L7-L28
https://github.com/heroku/buildpacks-python/blob/de378063c83e1eec6302e79f93f189162b5bbbbf/tests/python_version_test.rs#L172-L231